projects/ngx-datatable/src/lib/themes/material.scss

```
@@ -66,6 +66,11 @@ $datatable-pager-active-background: rgba(158, 158, 158, 0.2) !default;
 $datatable-summary-row-background: #ddd !default;
 $datatable-summary-row-background-hover: #ddd !default;
 
+$datatable-ghost-loader-background-light:  #f0f0f0;
+$datatable-ghost-loader-background:  #d8d6d6;
+$datatable-ghost-loader-background-dark:  #ccc7c7;
+
```

- Minor spacing adjustment.
- Where did you derive those colors from? Might make sense to sty internally consistent and re-use already existing colors, so the adoption to another design-system is easier.

```suggestion
$datatable-ghost-loader-background-light:  #f5f5f5;
$datatable-ghost-loader-background:  #d9d8d9;
$datatable-ghost-loader-background-dark:  #ccc7c7;
```